docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -1,23 +1,183 @@
-```
-components.*.spring.profiles.active=attls
-components.*.server.ssl.enabled=false
-components.*.server.internal.ssl.enabled=false
-```
-While API ML can not handle TLS on its own, the API Mediation Layer needs information about the server certificate that is defined in the AT-TLS rule. Update the `zowe.yaml` file for each respective API ML component in the `components` sections with the path to the SAF Key ring from the AT-TLS rule and specify the alias that is used for Inbound communication:
+Starting with Zowe version 2.13 it is possible to leverage AT-TLS within the API Mediation Layer. Each API ML component can run with AT-TLS rules applied. Some components, such as the Discovery service, can be made AT-TLS aware by enabling the AT-TLS profile, whereby TLS information can be utilized. Such information could be a client certificate.
+
+## Zowe configuration
+
+:::note
+Support for AT-TLS was introduced back in Zowe 1.24, however there was an issue that prevented startup in some versions. It is recommended to upgrade to 2.13 or newer for full support.
+:::
+
+To enable the AT-TLS profile and disable the TLS application in API ML, update `zowe.yaml` with following values under the respective component in the `zowe.components` section.
+
+```yaml
+components.*.spring.profiles.active: attls
+components.*.server.ssl.enabled: false
+components.*.server.internal.ssl.enabled: false
 ```
-components.*.certificate.keystore.file=<SAF-key-ring-from-AT-TLS-rule>
-components.*.certificate.keystore.type=JCERACFKS
-components.*.certificate.keystore.password=<keyring-password>
-components.*.certificate.keystore.alias=<certificate-alias-from-AT-TLS-rule>
+
+While API ML will not handle TLS on its own with AT-TLS enabled, the Mediation Layer needs information about the server certificate that is defined in the AT-TLS rule.
+
+Update the `zowe.yaml` file for each respective APIML component in the `zowe.components` sections with the path to the SAF Key ring from the AT-TLS rule and specify the alias that is used for Inbound communication:
+
+```yaml
+components.*.certificate.keystore.file: <SAF-key-ring-from-AT-TLS-rule>
+components.*.certificate.keystore.type: JCERACFKS
+components.*.certificate.keystore.password: password
+components.*.certificate.keystore.alias: <certificate alias / label from AT-TLS rule>
 ```
+
+Finally, if there is an outbound AT-TLS rule configured for the link between the API Gateway and z/OSMF, update or set the `zowe.zOSMF.scheme` to `http`.
+
 :::note
-This procedure does not configure AT-TLS on z/OS, but rather enables API ML to work with AT-TLS in place.
+AT-TLS is not yet supported in the API Cloud Gateway Mediation Layer component.
```

Comment:
What about Zlux? Could we mention that Zlux supports AT-TLS (within a link), but not in aware mode and describe the limitation such as nobody verifying communication...